galette/templates/default/pages/scheduledpayments_list.html.twig

```diff
@@ -270,9 +281,9 @@
                         </i>
                         <span class="ui special popup">
                             {% if contribution.isFee() %}
-                                {{ _T("Membership fee %id")|replace({"%id": contribution.id}) }}
+                                {{ _T("Edit related membership fee") }}
                             {% else %}
-                                {{ _T("Donation %id")|replace({"%id": contribution.id}) }}
+                                {{ _T("Edit related donation") }}
```

Comment:
Seems like there is no existing translation that can be used; but I'm not fan of "related", maybe just:
```suggestion
                                {{ _T("Edit donation") }}
```